varnish-mixin/README.md

```diff
@@ -0,0 +1,117 @@
+# Varnish Cache mixin
+
+Varnish Cache mixin contains a configurable dashboard and set of alerts.
+
+The Varnish Cache mixin contains the following dashboards:
+
+- Varnish overview
+
+and the following alerts:
+
+- VarnishCacheLowCacheHitRate
+- VarnishCacheHighMemoryUsage
+- VarnishCacheHighCacheEvictionRate
+- VarnishCacheHighSaturation
+- VarnishCacheSessionsDropping
+- VarnishCacheBackendFailure
+- VarnishCacheBackendUnhealthy
+
+## Varnish Cache overview
+
+The Varnish overview dashboard delivers a breakdown of metrics such as thread utilization, client and backend requests + connections, cache, network throughput, and client and backend log data, offering an in-depth perspective on connections and communication. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance.
+
+![First screenshot of the Varnish Cache overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/varnish-cache/screenshots/varnish_cache_overview_1.png)
+![Second screenshot of the Varnish Cache overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/varnish-cache/screenshots/varnish_cache_overview_2.png)
+
+In order to get logs, users must enable the `varnishncsa` process with the below commands after creating their varnish instance.
+
+```bash
+sudo systemctl enable varnishncsa
+sudo systemctl start varnishncsa`
+```
+
+Users have the choice of configuring their varnish logs to show whatever they'd like with the formatting flag of varnishncsa, by default this dashboard provides the following configuration.
+
+```bash
+sudo varnishncsa -F '{"Timestamp": "%t", "Varnish-Side": "%{Varnish:side}x", "Age": %{age}o, "Handling": "%{Varnish:handling}x", "Request": "%r", "Status": "%s", "Response-Reason": "%{VSL:RespReason}x", "Fetch-Error": "%{VSL:FetchError}x", "X-Forwarded-For": "%{x-forwarded-for}i", "Remote-User": "%u", "Bytes": "%b", "Time-To-Serve": %D, "User-Agent": "%{User-agent}i", "Referer": "%{Referer}i", "X-Varnish": "%{x-varnish}o", "X-Magento-Tags": "%{x-magento-tags}o"}}' -D -w /var/log/varnish/varnishncsa.log
+sudo varnishncsa -b -F '{"Timestamp": "%t", "Varnish-Side": "%{Varnish:side}x", "Handling": "%{Varnish:handling}x", "Request": "%r", "Status": "%s", "Response-Reason": "%{VSL:RespReason}x", "Fetch-Error": "%{VSL:FetchError}x", "Bytes": "%b", "Time-To-Serve": %D}' -D -w /var/log/varnish/varnishncsa-backend.log
+```
+
+The above two commmands provide client and backend logs. Note the
```

Comment:
Normally the flow is to readers read from top to bottom and not have readers backtrack to looking `above`. 

I would combine `Note the` with the explanation then give the example code.